eessi/testsuite/eessi_mixin.py

```diff
@@ -165,3 +175,33 @@ def assign_tasks_per_compute_unit(self):
     def request_mem(self):
         """Call hook to request the required amount of memory per node"""
         hooks.req_memory_per_node(self, app_mem_req=self.required_mem_per_node())
+
+    @run_after('setup')
+    def log_runtime_info(self):
+        """Log additional runtime information: which CVMFS repo was used (or if it was testing local software),
+        path to the modulefile, EESSI software subdir, EESSI testsuite version"""
+        self.postrun_cmds.append('echo "EESSI_CVMFS_REPO: $EESSI_CVMFS_REPO"')
+        self.postrun_cmds.append('echo "EESSI_SOFTWARE_SUBDIR: $EESSI_SOFTWARE_SUBDIR"')
+        if self.module_name:
+            # Get full modulepath
+            get_full_modpath = f'echo "FULL_MODULEPATH: $(module --location show {self.module_name})"'
+            self.postrun_cmds.append(get_full_modpath)
+
+    @run_after('run')
+    def extract_runtime_info_from_log(self):
+        """Extracts the printed runtime info from the job log and logs it as reframe variables"""
```

Comment:
should not run if doing a dry-run:
```suggestion
        """Extracts the printed runtime info from the job log and logs it as reframe variables"""
        if self.is_dry_run():
            return

```